Sources/FoundationEssentials/FileManager/FileManager+Basics.swift

```diff
@@ -60,6 +82,134 @@ internal struct _FileManagerImpl {
         atPath path: String,
         andPath other: String
     ) -> Bool {
+#if os(Windows)
+        return (try? path.withNTPathRepresentation {
+            let hLHS = CreateFileW($0, GENERIC_READ, FILE_SHARE_READ | FILE_SHARE_WRITE | FILE_SHARE_DELETE, nil, OPEN_EXISTING, 0, nil)
+            if hLHS == INVALID_HANDLE_VALUE {
+                return false
+            }
+            defer { CloseHandle(hLHS) }
+
+            return (try? other.withNTPathRepresentation {
+                let hRHS = CreateFileW($0, GENERIC_READ, FILE_SHARE_READ | FILE_SHARE_WRITE | FILE_SHARE_DELETE, nil, OPEN_EXISTING, 0, nil)
+                if hRHS == INVALID_HANDLE_VALUE {
+                    return false
+                }
+                defer { CloseHandle(hRHS) }
+
+                let dwLHSFileType: DWORD = GetFileType(hLHS)
+                let dwRHSFileType: DWORD = GetFileType(hRHS)
+
+                guard dwLHSFileType == FILE_TYPE_DISK, dwRHSFileType == FILE_TYPE_DISK else {
+                    return CompareObjectHandles(hLHS, hRHS)
+                }
+
+                var fiLHS: FILE_ID_INFO = .init()
+                guard GetFileInformationByHandleEx(hLHS, FileIdInfo, &fiLHS, DWORD(MemoryLayout.size(ofValue: fiLHS))) else {
+                    return false
+                }
+
+                var fiRHS: FILE_ID_INFO = .init()
+                guard GetFileInformationByHandleEx(hRHS, FileIdInfo, &fiRHS, DWORD(MemoryLayout.size(ofValue: fiRHS))) else {
+                    return false
+                }
+
+                if fiLHS.VolumeSerialNumber == fiRHS.VolumeSerialNumber, fiLHS.FileId == fiRHS.FileId {
+                    return true
+                }
+
+                var fbiLHS: FILE_BASIC_INFO = .init()
+                guard GetFileInformationByHandleEx(hLHS, FileBasicInfo, &fbiLHS, DWORD(MemoryLayout.size(ofValue: fbiLHS))) else {
+                    return false
+                }
+
+                var fbiRHS: FILE_BASIC_INFO = .init()
+                guard GetFileInformationByHandleEx(hRHS, FileBasicInfo, &fbiRHS, DWORD(MemoryLayout.size(ofValue: fbiRHS))) else {
+                    return false
+                }
+
+                if fbiLHS.FileAttributes & FILE_ATTRIBUTE_REPARSE_POINT == FILE_ATTRIBUTE_REPARSE_POINT,
+                   fbiRHS.FileAttributes & FILE_ATTRIBUTE_REPARSE_POINT == FILE_ATTRIBUTE_REPARSE_POINT {
+                    return (try? destinationOfSymbolicLink(atPath: path) == destinationOfSymbolicLink(atPath: other)) ?? false
```

Comment:
```suggestion
                    return (try? fileManager.destinationOfSymbolicLink(atPath: path) == fileManager.destinationOfSymbolicLink(atPath: other)) ?? false
```

This will allow deferring to subclasses when present

Reply:
Looks like this one still needs to be accounted for

Reply:
I think this change is still missing, but other than that this looks good to me!

Reply:
:sigh: I think that I might've applied it and then lost it. Thanks for the diligence in the review!